packages/esm-hiv-app/src/views/hiv-testing-services/tab-list/pre-test.component.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { EncounterListColumn, getObsFromEncounter, EncounterList } from '@ohri/openmrs-esm-ohri-commons-lib';
+import {
+  PreTestApproachConceptUUID,
+  PreTestConceptUUID,
+  PreTestEligibleForTestingConceptUUID,
+  PreTestEncounterTypeConceptUUID,
+  PreTestHIVTestDoneConceptUUID,
+  PreTestPopulationConceptUUID,
+} from '../../../constants';
+import { moduleName } from '../../../index';
+
+interface HIVPreTestTabListProps {
+  patientUuid: string;
+}
+
+const HIVPreTestTabList: React.FC<HIVPreTestTabListProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+
+  const columns: EncounterListColumn[] = useMemo(
+    () => [
+      {
+        key: 'vlDate',
+        header: t('PreTestEncDate', 'Encounter Date'),
```

Comment:
The key and translation are mis-aligned. Please let's match this to avoid any unnecessary translation duplications

Reply:
@pirupius Done